code/stable/glassbr/SRS/GlassBR_SRS.tex

```diff
@@ -93,13 +93,13 @@ \subsection{Table of Symbols}
 \\
 $interpZ$ & InterpZ & --
 \\
-$is-safeLoad$ & Variable that is assigned true when load resistance (capacity) is greater than applied load (demand) & --
+$is-safeLoad$ & Load resistance safety requirement & --
 \\
-$is-safeLR$ & Variable that is assigned true when load resistance (capacity) is greater than load (demand) & --
+$is-safeLR$ & Load resistance safety requirement & --
 \\
-$is-safePb$ & Variable that is assigned true when calculated probability is less than tolerable probability & --
+$is-safePb$ & Probability of breakage safety requirement & --
```

Comment:
Can we change this to "Probability of glass breakage ..."?  As mentioned above, it is nice to distinguish the general notion from the GlassBR specific.